solutions/usaco-894.mdx

```diff
@@ -20,6 +20,43 @@ $\max(\texttt{deg}[i])+1$ colors.
 
 <LanguageSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+// USACO-style file input
+void setIO(string name = ""){ // FastIO see General -> Fast Input and Output
+	ios_base::sync_with_stdio(0); cin.tie(0);
+	if(name.size()){
+		freopen((name+".in").c_str(), "r", stdin);
+		freopen((name+".out").c_str(), "w", stdout);
+	}
+}
+
+int main() {
+	setIO("planting");
+
+	int n; cin >> n;
+	int deg[n+1];
+	memset(deg, 0, sizeof(deg)); // Initialize all degrees to 0
```

Comment:
think you can do this w/ list initialization or smth
like `int deg[n + 1] {0}`
right @maggie-j-liu?

Reply:
also whitespace between ops here

Reply:
`int deg[n + 1]{};` should work, but change the array to a constant size. variable length arrays are not c++ standard